webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -147,20 +147,45 @@ const loadJobs = () => {
       if (data.error) {
         alert(data.message);
       } else {
-        $('#view-table').html(jobTableComponent({
-          jobs: data.slice(0, 300),
-          getDurationInSeconds,
-          convertTime,
-          convertState,
-        }));
+        let displayDataSet = [];
+        let rowCount = limit ? limit : 1000;
+        for (let i = 0; i < data.slice(0, rowCount).length; i++) {
+          let vcName = (data[i].virtualCluster) ? data[i].virtualCluster : 'default';
+          let stopBtnStyle = (data[i].executionType === "STOP" || data[i].subState === "FRAMEWORK_COMPLETED") ? '<button class="btn btn-default btn-sm" disabled>Stop</button>' : '<button class="btn btn-default btn-sm" onclick="stopJob(\'' + data[i].name + '\')">Stop</button>'
+          displayDataSet.push({
+            JobName: '<a href="view.html?jobName=' + data[i].name + '\">' + data[i].name + '</a>',
+            userName: data[i].username,
+            vcName: vcName,
+            StartTime: '<span title=\"' + Math.round(data[i].createdTime / 1000) + '\"/>' +
+              convertTime(false, data[i].createdTime),
+            Duration: '<span title=\"' + getDurationInSeconds(data[i].createdTime, data[i].completedTime) + '\"/>' +
+              convertTime(true, data[i].createdTime, data[i].completedTime),
+            Retries: data[i].retries,
+            Status: convertState(data[i].state),
+            Stop: stopBtnStyle
+          })
+        }
+        $('#view-table').html(jobTableComponent({}));
         table = $('#job-table').dataTable({
+          data: displayDataSet,
+          columns: [
+            { title: 'JobName', data: 'JobName' },
+            { title: 'userName', data: 'userName' },
+            { title: 'vcName', data: 'vcName' },
+            { title: 'StartTime', data: 'StartTime' },
+            { title: 'Duration', data: 'Duration' },
+            { title: 'Retries', data: 'Retries' },
+            { title: 'Status', data: 'Status' },
+            { title: 'Stop', data: 'Stop' }
+          ],
           'scrollY': (($(window).height() - 265)) + 'px',
           'lengthMenu': [[20, 50, 100, -1], [20, 50, 100, 'All']],
-          'order': [[2, 'desc']],
+          'order': [[3, 'desc']],
           'columnDefs': [
-            {type: 'natural', targets: [0, 1, 4, 5]},
-            {type: 'title-numeric', targets: [2, 3]},
+            { type: 'natural', targets: [0, 1, 2, 5, 6] },
+            { type: 'title-numeric', targets: [3, 4] },
```

Comment:
`object-curly-spacing` in `npm run lint`.
I'm not sure why this pass the tests, this branch failed on my local `npm test`.